test/receiverSpec.js

```diff
@@ -478,6 +483,19 @@ describe('Receiver Module', function () {
           expect(results).to.deep.equal(['sent'])
         })
 
+        it('should not fulfill payment if reviewPayment callback rejects', function * () {
```

Comment:
This needs a test to make sure that if you instantiate what you think is just an IPR receiver you can't accidentally get money showing up in your account with KEP